_data/khou22/khou22.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Kevin Hou
+institution: Princeton University
+profile_pic: kevin.png # Name and extension of your profile picture(ex. mona.png)
+quote: "We didn't realize we were making memories, we just knew we were having fun." -Winnie the Pooh
```

Comment:
Remove double quotes from begin and end.

Reply:
Done, thank you!